x-pack/plugins/cloud_integrations/cloud_chat/public/components/index.tsx

```diff
@@ -5,22 +5,5 @@
  * 2.0.
  */
 
-import React, { Suspense } from 'react';
-import { EuiErrorBoundary } from '@elastic/eui';
-
-/**
- * A suspense-compatible version of the Chat component.
- */
-export const LazyChat = React.lazy(() => import('./chat').then(({ Chat }) => ({ default: Chat })));
-
-/**
- * A lazily-loaded component that will display a trigger that will allow the user to chat with a
- * human operator when the service is enabled; otherwise, it renders nothing.
- */
-export const Chat = () => (
-  <EuiErrorBoundary>
-    <Suspense fallback={<div />}>
-      <LazyChat />
-    </Suspense>
-  </EuiErrorBoundary>
-);
+export { Chat, type ChatApi, type Props as ChatProps } from './chat';
+export { ChatHeaderMenuItem } from './chat_header_menu_item';
```

Comment:
nit: I would *really prefer* these new components be in a package.

Reply:
Got it, sure, let's discuss